e2e/test/iothub/DeviceClientX509AuthenticationE2ETests.cs

```diff
@@ -141,6 +143,77 @@ public async Task X509_Enable_CertificateRevocationCheck_Amqp_WebSocket()
             await SendMessageTest(transportSetting).ConfigureAwait(false);
         }
 
+        [LoggedTestMethod]
+        public async Task X509_Cert_Chain_Install_Test_MQTT()
```

Comment:
small nit: you might want to rename the test method to say "mqtt_tcp", since "mqtt" by itself is usually interpreted in our test project to be "tcp with fallback to ws".

Reply:
I see. Will update.